src/test/groovy/com/ofg/geb/pages/SwaggerUIHomePage.groovy

```diff
@@ -0,0 +1,28 @@
+package com.ofg.geb.pages
+
+import geb.Page
+
+class SwaggerUIHomePage extends Page {
+    static url = "http://localhost:8095/swagger"
```

Comment:
Page file has now only '/swagger', rest is in 'baseUrl' property in GebConfig.groovy